annotations/component-annotations/src/main/java/io/ap4k/component/handler/ResourceGroup.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ * <p>
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
```

Comment:
EPL?

Reply:
What do you mean?

Reply:
what do you mean?

Reply:
The license header is wrong. the project is license as Apache.

Reply:
Hmm, not sure where the hell that's coming from… 🤔